### DISCUSSION TITLE
timeline for Multi-node / multi-gpu training for RSL_RL

### DISCUSSION THREAD

**Discussion by dxyy1** [isaac-sim/IsaacLab#1543] [Q&A]
Hi there,

I was wondering if there is a rough timeline when we can expect multi-gpu training been brought to rsl_rl ? If it is still a while away, I was wondering if there is any useful resources or instructions on how to implement it, so I can try and do it myself?

I have watched the DDP in PyTorch tutorial series on YT. But it didn't cover how to spawn different envs on different GPUs, so I was wondering what is the general steps to enabling multi-gpu training for RL too.

Thanks,

**Comment by RandomOakForest** [comment]
Thank you for posting this. It is not clear when this feature will be implemented. I've asked the team about guidelines to implement it and will be posting them here.

**Reply by dxyy1** [reply]
Thanks! Looking forward to learning about implementing the multigpu pipeline for rls_rl.

**Reply by dxyy1** [reply]
hi there, is there any updates on this?

**Reply by abhiyanthrik** [reply]
We also are waiting for the same. Is there any update??
